magprime/automated_emails.py

```diff
@@ -109,10 +109,6 @@ def __init__(self, event):
                          a.gets_any_kind_of_shirt, when=before(c.SHIRT_DEADLINE),
                ident='magprime_shirt_size_confirmation')
 
-AutomatedEmail(Attendee, 'MAGFest Dealer waitlist has been exhausted', 'dealer_waitlist_exhausted.txt',
```

Comment:
Nothing in the code would ever add this phrase to the admin notes -- this was likely done via a script one year.